### DISCUSSION TITLE
ROS2 Progress?

### DISCUSSION THREAD

**Discussion by JurrijnP** [ros-industrial/abb_robot_driver#49] [Q&A] [answered]
I was wondering if there is any progress in porting this package to ROS2?

I am currently working on a project where I plan to use this package. I have no problems with using ROS1 for now as I understand porting this package to ROS2 is not the easiest.

But in terms of future development with ROS2 and an ABB robot this package seems to be exactly what is needed for the project. So if anybody has an answer to my question it would be greatly appreciated so I can write it down as future recommendation :)

**Comment by JurrijnP** [comment]
friendly 🛎️

**Comment by gavanderhoorn** [comment, marked as answer]
I cannot speak "for ABB" nor for ABB Corporate Research.

Personally, I do not have plans to port this to ROS 2.

There is a community effort to create a `ros2_control` based driver here: [PickNikRobotics/abb_ros2](https://github.com/PickNikRobotics/abb_ros2).

---

Edit: note: my comment should not be interpreted as an endorsement or recommendation of any particular package(s). I merely wanted to make you aware of their existence.
